sdk/keyvault/keyvault-certificates/samples/javascript/backupAndRestore.js

```diff
@@ -23,7 +23,11 @@ async function main() {
   const url = process.env["KEYVAULT_URI"] || "<keyvault-url>";
   const credential = new DefaultAzureCredential();
 
-  const client = new CertificateClient(url, credential);
+  const client = new CertificateClient(url, credential, {
+    // The KEYVAULT_API_VERSION environment variable is used by our CI pipelines to run the samples and check their validity automatically.
+    // The serviceVersion is an optional parameter that allows users to specify a Key Vault service API version.
+    serviceVersion: process.env.KEYVAULT_API_VERSION
```

Comment:
Shouldn't you default this in case it's not passed for whatever reason? Or does it's absense (`undefined`?) mean that the client will default it itself?

Reply:
If it's undefined, the client uses the LATEST_API_VERSION: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/keyvault/keyvault-keys/src/index.ts#L251